lblod/frontend-form-builder
# SFB-196: add search input field for the form overview

## ID
 [SFB-196](https://binnenland.atlassian.net/browse/SFB-196)

 ## Description

As a user I want to search my form by name. this is done by using the `mu-cl resource` filters.

 ## Type of change


 ## How to test

1. go to the form overview
2. a search input should be visible
3. when typing in the search input the data will update to the requested values

 ## Links to other PR's

 - /
